dashboard/src/components/api/workspace/che-workspace.factory.ts

```diff
@@ -43,24 +40,38 @@ export enum WorkspaceStatus {
   ERROR
 }
 
+type workspacesEvent = 'onChangeWorkspaces' | 'onDeleteWorkspace';
+
+const MAX_ITEMS = 256;
```

Comment:
It fixes the issue with `more than 30 workspaces is not displayed`, but it's easy to imagine new one `more than 256 workspaces are not displayed` )
It would be better to solve an issue in general and fetch all workspaces page by page. But I'm OK with proposed change as an easy fix, and probably no one will create more workspaces than 256.

Reply:
Thanks